test/msgpax/ext/overwrite_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule Msgpax.Ext.OverwriteTest do
+  use Msgpax.Case, async: false
```

Comment:
These tests need to be the last one to be run so that the overwrites don't affect the other tests. Hence the `async: false`, as `ExUnit` first loads `async: true` tests.